### DISCUSSION TITLE
Request for feedback: DataCite Public Data File

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#133] [Requests for Comment]
In March 2024, we launched the [first DataCite Public Data File](https://datacite.org/blog/announcing-datacites-first-public-data-file/), and we’d love to hear your feedback as we start to prepare next year’s file.

## About the Public Data File

The Data File contains JSON metadata for all publicly available DataCite DOIs that were registered up to the end of 2023 in a single compressed file. 

Explore the 2023 file by:

- Accessing the file [here](https://datafiles.datacite.org/datafiles/public-2023)  
- Consulting [our support documentation](https://support.datacite.org/docs/datacite-public-data-file)

## Providing feedback

We’re excited to hear your thoughts on the 2023 file and ways to enhance future iterations. Contribute your feedback by leaving a comment on this post. You are welcome to reply by answering any or all of the questions below: 

* How are you using the 2023 DataCite Public Data File?   
* What would you change about the file or the service? 

> [!NOTE]
> We are also exploring a monthly data file snapshot service to complement the annual data file offering. You can read about this proposal and share feedback here: https://github.com/datacite/datacite-suggestions/discussions/134.

_This discussion will close on 13 December 2024, but you can still submit feedback on [DataCite Suggestions](https://github.com/datacite/datacite-suggestions/discussions/67)._

**Comment by dspinellis** [comment]
The [Alexandria3d](https://github.com/dspinellis/alexandria3k) Python library and command-line tool use the Crossref data dump to provide efficient relational query access to it and combined it with other data sources, such as ORCID, DOAJ, PbuMed, DataCite, and US patents. Several example use cases are included in an [accompanying paper](https://doi.org/10.1371/journal.pone.0294946) and on the software's GitHub repo [examples folder](https://github.com/dspinellis/alexandria3k/tree/main/examples). Monthly dumps would be awesome: among other uses, they would allow the dynamic construction of researcher and organization research output reports. Bringing back topic classification would also be nice, though I realize the limitations of the previously employed method.

**Reply by codycooperross** [reply]
Thank you for telling us about Alexandria3d, @dspinellis ! It's great to hear that the monthly data file service sounds useful to you. If you didn't already take a look at our proposed initial feature set, feel free here: https://github.com/datacite/datacite-suggestions/discussions/134 

Can you let us know which topic classification feature you're referring to?

**Reply by dspinellis** [reply]
I upvoted the monthly data file service with no comments on it. Regarding topic classification, I was referring to the [experimental addition of the ASJC Scopus codes](https://community.crossref.org/t/retrieve-subjects-and-subject-from-journals-and-works/2403).
